app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -68,74 +62,92 @@ import org.mozilla.fenix.wallpapers.Wallpaper
-                        )
-                    }
-                    onSelectWallpaper(updatedWallpaper)
-                },
-            )
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .fillMaxHeight()
+                .verticalScroll(rememberScrollState()),
+        ) {
+            wallpaperGroups.forEach { (collection, wallpapers) ->
+                if (wallpapers.isNotEmpty()) {
+                    WallpaperGroupHeading(
+                        onLearnMoreClick,
+                        collection,
+                    )
+                    WallpaperThumbnails(
+                        wallpapers = wallpapers,
+                        defaultWallpaper = defaultWallpaper,
+                        loadWallpaperResource = loadWallpaperResource,
+                        selectedWallpaper = selectedWallpaper,
+                        onSelectWallpaper = { updatedWallpaper ->
+                            onSelectWallpaper(updatedWallpaper)
+                        },
+                    )
+                }
+            }
         }
     }
 }
 
 @Composable
-private fun WallpaperSnackbar(
-    onViewWallpaper: () -> Unit,
+private fun WallpaperGroupHeading(
+    onLearnMoreClick: (String) -> Unit,
+    collection: Wallpaper.Collection,
 ) {
-    Snackbar(
-        modifier = Modifier
-            .padding(8.dp)
-            .heightIn(min = 48.dp),
-        backgroundColor = FirefoxTheme.colors.actionPrimary,
-        content = {
+    Column(
+        modifier = Modifier.padding(
+            top = 16.dp,
+            start = 18.dp,
+            end = 18.dp,
+            bottom = 12.dp,
+        ),
+    ) {
+        if (collection.name != "classic-firefox") {
             Text(
-                modifier = Modifier.padding(start = 16.dp, end = 16.dp, top = 8.dp, bottom = 8.dp),
-                text = stringResource(R.string.wallpaper_updated_snackbar_message),
-                textAlign = TextAlign.Start,
-                color = FirefoxTheme.colors.textOnColorPrimary,
-                overflow = TextOverflow.Ellipsis,
-                maxLines = 2,
-                style = FirefoxTheme.typography.headline7,
+                text = collection.heading ?: "Limited edition",
+                modifier = Modifier.padding(bottom = 2.dp),
+                color = FirefoxTheme.colors.textSecondary,
+                style = FirefoxTheme.typography.subtitle2,
             )
-        },
-        action = {
-            TextButton(
-                text = stringResource(R.string.wallpaper_updated_snackbar_action),
-                onClick = onViewWallpaper,
-                modifier = Modifier.padding(all = 8.dp),
-                textColor = FirefoxTheme.colors.textOnColorPrimary,
+
+            Row(modifier = Modifier.fillMaxWidth()) {
+                collection.description?.let {
+                    Text(
+                        text = "$it. ",
```

Comment:
It looks like UX is planning to include punctuation in the descriptions, so we can remove the period from the text parameter here.

Reply:
Actually, come to think of it I believe the intent for this release is to have the strings all included as resources. So we will  need a text resource for the description that is:
`The new Independent Voices collection.` and a text resource for the heading that is `Limited edition`. We can revisit doing this more dynamically in a follow-up to the MR. 

The text resources will be instead of using the properties defined on the `collection`.